src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -8,6 +8,10 @@ import OptionRowLHNData from './OptionRowLHNData';
 import variables from '../../styles/variables';
 
 const propTypes = {
+    /** Wrapper style for the section list */
+    // eslint-disable-next-line react/forbid-prop-types
+    style: PropTypes.arrayOf(PropTypes.object),
```

Comment:
not sure why it should be disabled eslint check - i think in styles you will have object and not array of objects